packages/components/socialFeed/SocialActions/TipModal.tsx

```diff
@@ -80,22 +109,18 @@ export const TipModal: React.FC<{
       return;
     }
 
-    const amount = Decimal.fromUserInput(
-      fieldValues.amount,
-      nativeCurrency.decimals,
-    ).atomics;
-
     if (selectedNetworkInfo?.kind === NetworkKind.Gno) {
```

Comment:
IMHO, we should handle this in a more consistent and generic way: 

We can have: useSocialFeedTipPostMutation({...})

In in that hook we switch for Teritori, Gno... and other networks later

Reply:
We could make an exported hook `useSocialFeedTipPostMutation` and handle Gno and Cosmos networks in it, yes.
But not in this PR

Reply:
We could make an exported hook `useSocialFeedTipPostMutation` and handle Gno and Cosmos networks in it, yes.
But not in this PR